src/cmdlib.sh

```diff
@@ -350,14 +356,11 @@ workdir=${workdir}
 # chmod will set ownership to root, not builder
 export USER=$(id -u)
 
-# ensure the user of files created do not have root ownership
-trap 'chown -h -R ${USER}:${USER} ${workdir}' EXIT
-
 $(cat "${DIR}"/supermin-init-prelude.sh)
 rc=0
 sh ${TMPDIR}/cmd.sh || rc=\$?
 echo \$rc > ${workdir}/tmp/rc
-/sbin/fstrim -v ${workdir}/cache
+${cachedisk+/sbin/fstrim -v ${workdir}/cache}
```

Comment:
This is a cool shell trick, but lets maybe do something like `[ -n "$cachedisk" ] && fstrim....` for clarity?

Reply:
We want this substitution to happen at the time the heredoc is defined though, not actually as part of the logic.

Reply:
Just expanded it to `[ -b /dev/vdb1 ]`!